lib/update-lockfile.js

```diff
@@ -32,7 +32,9 @@ module.exports = function updateLockfile (dependency, options) {
 
     if (options.yarn) {
       exec(`yarn add ${args}`)
-    } else {
+    }
+
+    if (options.npm) {
```

Comment:
I do disagree with you.  
It would already crash in line 18, since there is no validation performed that the passed in `options` object is not null or undefined. That means on the other hand !options will never become true, since `!{} // false`. Besides that in `update.js` I pass in the npm option based on the check if a `packageLock` was found or not. Is there any known use case that does not use `update.js` as the entry point an rather `lib/update-lockfile.js` directly?